app/components/post_textbox/post_textbox.js

```diff
@@ -170,16 +170,18 @@ export default class PostTextbox extends PureComponent {
     };
 
     handleContentSizeChange = (event) => {
-        let contentHeight = event.nativeEvent.contentSize.height;
-        if (contentHeight < INITIAL_HEIGHT) {
-            contentHeight = INITIAL_HEIGHT;
-        } else if (Platform.OS === 'ios') {
-            contentHeight += 5;
-        }
+        if (Platform.OS === 'android') {
+            let contentHeight = event.nativeEvent.contentSize.height;
+            if (contentHeight < INITIAL_HEIGHT) {
+                contentHeight = INITIAL_HEIGHT;
+            } else if (Platform.OS === 'ios') {
```

Comment:
Why is this now all wrapped to only run on Android? If that's correct, this else if case should be removed